src/ray/object_manager/object_manager.h

```diff
@@ -41,8 +41,11 @@ struct ObjectManagerConfig {
   uint64_t object_chunk_size;
   /// The stored socked name.
   std::string store_socket_name;
-  /// Maximun number of push retries.
-  int max_push_retries;
+  /// The time in milliseconds to wait until a Push request
+  /// fails due to unsatisfied local object. Special value:
+  /// Negative: waiting infinitely.
+  /// 0: failing directly.
```

Comment:
Does this mean: If the object is not local, then fail immediately? If so, can you update to read: "Fail immediately if the object is not local."

Reply:
"gives up" or "times out" would be a little clearer than "fails"